README.md

```diff
@@ -147,20 +143,7 @@ Usage:
 PYTHONPATH=./ python examples/run_action_recognition.py
 ```
 
-#### Demo 2: Gesture Control
-
-`examples/run_gesture_control.py` applies our pre-trained models to the detection of 8 hand gesture events 
-(6 swiping gestures + thumbs up + thumbs down). Compared to Demo 1, the model used in this case was trained 
-to trigger the correct class for a short period of time right after the hand gesture occurred. This behavior 
-policy makes it easier to quickly trigger multiple hand gestures in a row. 
-
-Usage:
-```shell
-PYTHONPATH=./ python examples/run_gesture_control.py
-```
-
-
-#### Demo 3: Fitness Activity Tracking
+#### Demo 2: Fitness Activity Tracking
```

Comment:
I wanted to leave the fitness tracking section next to the calorie estimation and repetition counting ones? Let me know what you think